src/Entity/Banner.php

```diff
@@ -423,30 +501,42 @@ public function toArray(): array
     {
         $data = [];
 
-        if($this->name !== null) {
+        if ($this->name !== null) {
```

Comment:
In the previous PR I saw you removed the if/else checks for properties. You can it here, too.